hass_client/Client.py

```diff
@@ -414,20 +414,19 @@ def batteryState(self, device):
   def publish_discovery(self, device, type, deviceId, config):
     base_topic = self.config('base_topic')
     device_name = self.config('device_name')
-    config.update({ 
+    config.update({
       'unique_id': '%s_%s' % (getMacAddr(), deviceId),
       'availability_topic': (
         '%s/%s/available' % (base_topic, device_name) if base_topic \
         else '%s/available' % device_name
       ),
       'device': {
-        'identifiers': getMacAddr(),
-        'connections': [['mac', getMacAddr(False)]],
-        'manufacturer': 'Telldus Technologies',
-        'model': Board.product(),
-        'name': device_name,
-        'sw_version': Board.firmwareVersion()
-      } 
+        'identifiers': device.getOrCreateUUID(),
+        'manufacturer': device.protocol().title(),
+        'model': device.model().title(),  # Model is always 'n/a' but is supposed to be updated.
+        'name': device.name(),
+        'via_hub': getMacAddr(),
```

Comment:
Ska det inte vara via_device ?
HA loggen klagade på att via_hub var depricated.

Reply:
Jo det stämmer.. via_device heter det..

```suggestion
        'via_device ': getMacAddr(),
```